fontir/src/ir.rs

```diff
@@ -41,32 +41,40 @@ pub struct StaticMetadata {
     /// See <https://learn.microsoft.com/en-us/typography/opentype/spec/name>.
     pub names: HashMap<NameKey, String>,
 
-    /// Every axis used by the font being compiled
+    /// Every axis used by the font being compiled, including point axes.
```

Comment:
I suggest using "discrete axes" as that's the name used in DesignSpace format

Reply:
wait, maybe is this about something else?

Reply:
"an axis of variation defined only at a single point"... It's not immediately clear to me why we need to special case these

Reply:
I see you want to exclude these so-called "point axes" when constructing the VariationModel. In reality, these kind of setups (a variaiton axis where min=default=max) do not actually occur in real world font sources because they do not make sense to have, they would be no op. They may occur in your made-up test data (e.g. static.designspace) but I don't think it is worth having to different `Vec<Axis>` in the static metadata, one for axes in general and one for actually variable ones (which is 100% of them in an actually variable font project). You can filter on demand when needed, no?

Reply:
FWIW fontmake does not treat these point-axes specially; e.g. if you feed your static.designspace to fontmake-py and ask it to make a "variable" font, `fontmake resources/testdata/static.designspace -o variable`, it will make one with fvar containing that point axis, and all the other variation tables basically empty.
If you ask it to create static ttfs from the same designspace (-o ttf), it will make one for each of the masters (or one for each of the instances with additional -i option), and would be equivalent to running fontmake directly on the master UFO as input. I'm not saying we should replicate this UX in fontmake-rs, I think the latter should be designed to build variable fonts only.
Just saying that a font like our test static.designspace does not normally occur in the wild; if one wants to build a static font, one builds that one, doesn't create a dummy designspace to wrap it.